From c258b8f1a13f755f0a79de309196cd34b4abba6e Mon Sep 17 00:00:00 2001
From: Sergey Fedorov <vital.had@gmail.com>
Date: Mon, 21 Jul 2025 01:20:21 +0800
Subject: [PATCH 18/31] MacBundles.gmk: bundles for 10.7+

This is a hack, we do not actually define macosx_lion_up.
---
 make/MacBundles.gmk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make/MacBundles.gmk b/make/MacBundles.gmk
index a4e7db333d..075ff24ae7 100644
--- a/make/MacBundles.gmk
+++ b/make/MacBundles.gmk
@@ -30,7 +30,7 @@ include TextFileProcessing.gmk
 default: bundles
 
 # Only macosx has bundles defined.
-ifeq ($(OPENJDK_TARGET_OS), macosx)
+ifeq ($(OPENJDK_TARGET_OS), macosx_lion_up)
 
   bundles: jre-bundle jdk-bundle
 
-- 
2.24.3 (Apple Git-128)

